docs/src/define-routes/https.md

```diff
@@ -53,13 +38,19 @@ That results in all pages being served over TLS, and any requests for an HTTP UR
     to: "https://{default}/"
 ```
 
-Specifying only HTTP routes results in duplicate HTTPS routes being created automatically,
-allowing the site to be served from both HTTP and HTTPS without redirects.
+All traffic to your domain is sent to your app. The `www` subdomain redirects to the default domain. This also includes redirecting requests from HTTP to HTTPS. It affects your [default domain](../define-routes/_index.md#default).
+
+[See more example on routes configuration](../define-routes/_index.md).
 
-Although Platform.sh doesn't recommend it, you can also redirect HTTPS requests to HTTP explicitly to serve the site over HTTP only.
+## Using HTTP only
+
+Platform.sh recommends using HTTPS requests for all sites exclusively (although both HTTP and HTTPS are supported).
+Using only HTTPS provides better security and access to certain features that web browsers only permit over HTTPS such as HTTP/2 connections, which can greatly improve performance.
+
+Although Platform.sh doesn't recommend it, you can redirect HTTPS requests to HTTP explicitly to serve the site over HTTP only.
 The use cases for this configuration are few.
```

Comment:
Hmmm, I could imagine valid uses cases for legacy apps (although it's an edge case and probably not a good thing at all to use only http), I would leave it there since we also still allow using very outdated frameworks and languages.
But either way is fine I suppose.